internal/monitors/cadvisor/metadata.yaml

```diff
@@ -232,4 +232,13 @@ monitors:
     This monitor pulls cadvisor metrics through a
     Kubernetes kubelet instance via the `/stats/container` endpoint.
   monitorType: kubelet-stats
+  metrics:
+    pod_ephemeral_storage_capacity_bytes:
+      description: Represents the storage space available (bytes) for the filesystem
+      default: false
+      type: gauge
+    pod_ephemeral_storage_used_bytes:
+      description: Represents the bytes used on the filesystem
+      default: false
+      type: cumulative
```

Comment:
This seems more like a gauge to me since it can go up and down over time.

Reply:
Good point, updated to gauge.